content.js

```diff
@@ -40,3 +40,4 @@ if (PACER.hasPacerCookie(document.cookie)) {
 } else {
   console.info(`RECAP: Taking no actions because not logged in: `+url);
 }
+
```

Comment:
Spurrious newline.

Reply:
This is now part of the editor config. They'll be appearing...

Reply:
That's unfortunate, it screws up all kinds of `git blame` operations. If you want to do it, please make sure it's a separate commit that doesn't contain any functional changes (not squashed with or integrated with the rest).

Or go through and touch all files at once. (Disfavored. But not as much as having it happen randomly.)

But this is the reason that such editor configuration automatic linting/KNFing/whatever are generally disfavored. They produce more churn in the code that makes it harder for other developers and future people trying to find the history of a change.

But also:
Why is this happening on this file that is not otherwise being touched in this changset?
Why would you want to have an EXTRA blank line at the end of a file. It's a reasonable style rule to have make sure that files end in a final newline (cf. `require-final-newline t`), but not a common one to require they end in **two**.

Reply:
> Why is this happening on this file that is not otherwise being touched in this changset?

Only because I had monkeyed with it at first, then didn't catch that I left extra newlines on there. So this one is on me.